src/OrleansRuntime/Silo/Silo.cs

```diff
@@ -1054,6 +1038,16 @@ internal void SetMaxForwardCount_ForTesting(int val)
                 silo.OrleansConfig.Globals.MaxForwardCount = val;
             }
 
+            internal void LatchIsOverloaded(bool overloaded)
+            {
+                this.silo.Metrics.LatchIsOverload(overloaded);
```

Comment:
K, let's merge as is then. We can decide to fix later if it doesn't bring braking changes